test/uk/gov/hmrc/plasticpackagingtax/registration/controllers/ContactDetailsConfirmAddressControllerSpec.scala

```diff
@@ -294,6 +320,17 @@ class ContactDetailsConfirmAddressControllerSpec extends ControllerSpec {
 
           intercept[RuntimeException](status(result))
         }
+
+        "user submits form and incorpJourneyId is not present" in {
+          authorizedUser()
+          mockRegistrationFind(aRegistration(withIncorpJourneyId(incorpJourneyId = None)))
+          mockRegistrationException()
+
+          val correctForm = Seq("useRegisteredAddress" -> "yes", formAction)
+          val result      = controller.submit()(postJsonRequestEncoded(correctForm: _*))
+
+          redirectLocation(result) mustBe Some(routes.RegistrationController.displayPage().url)
+        }
```

Comment:
Is journey id really necessary now that we are taking details from our cache/db?

Reply:
removed the test